rust/src/domains/polars/expr/mod.rs

```diff
@@ -65,3 +65,12 @@ impl<D: DatasetMetric> MetricSpace for (ExprDomain, D) {
         return true;
     }
 }
+
+impl ExprDomain {
+    pub fn active_column(&self) -> Fallible<String> {
+        return self
+            .active_column
+            .clone()
```

Comment:
The reason why `.cloned()` didn't work here was because `self.active_column` is of type `&Option<String>` not `Option<&String>`. My bad!